src/ui/preferences/GeneralSettings.qml

```diff
@@ -198,18 +198,27 @@ QGCView {
                                 enabled:            true
                             }
                         }
+                        Row {
+                            spacing: ScreenTools.defaultFontPixelWidth
+                            visible: offlineVehicleCombo.currentText == "Sub"
```

Comment:
I chose to put the checkbox in the offline section because then it could appear based on which vehicle is selected so that it doesn't affect anything other than Sub. I could also put this in the main settings section, but it seemed inappropriate. Thoughts?